src/constructs/ec2/vpc.test.ts

```diff
@@ -0,0 +1,43 @@
+import { SynthUtils } from "@aws-cdk/assert";
+import type { SynthedStack } from "../../../test/utils";
+import { simpleGuStackForTesting } from "../../../test/utils";
+import { GuVpc } from "./vpc";
+
+describe("The GuVpc class", () => {
+  describe("subnets method", () => {
+    test("returns an array of subnets with the correct ids", () => {
+      const stack = simpleGuStackForTesting();
+
+      const subnets = GuVpc.subnets(stack, ["subnet1", "subnet2"]);
+
+      expect(subnets[0].subnetId).toBe("subnet1");
+      expect(subnets[1].subnetId).toBe("subnet2");
+    });
```

Comment:
minor: destructuring might read better here and also make it easier to assert against the length of the array?

```suggestion
      const [subnet1, subnet2, ...rest] = GuVpc.subnets(stack, ["subnet1", "subnet2"]);

      expect(subnet1.subnetId).toBe("subnet1");
      expect(subnet2.subnetId).toBe("subnet2");
      
      // test that no extra subnets are defined
      expect(rest.length).toBe(0);
    });
```

Reply:
Nice 👍